src/domain/entities/DataSetMetadata.ts

```diff
@@ -0,0 +1,5 @@
+import { DataSet } from "./DataSet";
+
+export interface DataSetMetadata {
```

Comment:
[subjective] Typically, we group very related interfaces in the "parent" entity (`DataSet`), but it's ok like this.

Reply:
Ok, wasn't sure if it was better to keep them separated. Moved to entities/DataSet